LICENSE.txt

```diff
@@ -1844,3 +1844,33 @@ This project includes code from the autobrew project.
 Copyright: Copyright (c) 2017 - 2019, Jeroen Ooms.
 All rights reserved.
 Homepage: https://github.com/jeroen/autobrew
+
+--------------------------------------------------------------------------------
+
+The files under dev/tasks/homebrew-formulae have the following license:
```

Comment:
Could you move `ci/apache-arrow.rb` to `dev/tasks/homebrew-formulae/apache-arrow.rb` and refer only `dev/tasks/homebrew-formulae/*.rb`? `dev/tasks/homebrew-formulae/travis.osx.yml` is out of scope.

Reply:
Sorry, you're right, will fix.

I worry that `dev/tasks/homebrew-formulae/apache-arrow.rb` is too obscured; it's nice to be able to find build scripts more easily and `ci/` seems to be where most of them are (or maybe that's not right either).